content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2239,7 +2295,7 @@ Introspection and Prisma Migrate do not currently support MongoDB, which means P
 | Name     | Required | Type               | Description                                                                                                                                                                                                                                 |
 | -------- | -------- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `fields` | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]`                                                                                                                                                                            |
-| `name`   | **No**   | `name`             | The name of the index in the underlying database (Prisma generates an index name that respects identifier length limits if you do not specify a name. Prisma uses the following naming convention: `tablename.field1_field2_field3_unique`) |
```

Comment:
there is still a name argument, but it refers to the name in the client api, see @@id